sql/updates/latest-dev.sql

```diff
@@ -2,3 +2,21 @@
 -- we add an addition optional argument to locf
 DROP FUNCTION IF EXISTS locf(ANYELEMENT,ANYELEMENT);
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.finalize_agg_sfunc(
+tstate internal, aggfn TEXT, inner_agg_collation_schema NAME, inner_agg_collation_name NAME, inner_agg_serialized_state BYTEA, return_type_dummy_val ANYELEMENT)
```

Comment:
why is aggfn not REGPROC?

Reply:
because that's not safe for dump/restore.